django/static/overwolf-logo.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 326 332" xmlns="http://www.w3.org/2000/svg">
+    <rect width="100%" height="100%" rx="10%" fill="white"/>
```

Comment:
I feel like this is the inverse of [the current brand guidelines](https://www.overwolf.com/brand-guidelines/), as in, the icon itself should be white and the area around it dark. Also the other provider icons are a single color whereas this uses two now, which is a bit odd.

I'd suggest just removing the rectangle here entirely and changing the icon fill color to white, that should probably match much better?

Reply:
Seems I forgot to include a screenshot. The color scheme was chosen to match the current icons, where the box matches text color and the "content" matches the background color.

![image](https://user-images.githubusercontent.com/3500016/229513159-66e289c5-e9cf-4488-b950-f3eb2c2e4bcb.png)

Reply:
I'd still suggest swapping it the other way around, I'm pretty sure the current ones aren't like that due to consistency, but just because they used to be presented like that by the companies themselves

Reply:
Well if we want to stick to OW brand guidelines, do you suggest using the "Overwolf" text from SVG under the icon or next to in? Do we keep the text label anyway?

Reply:
I'd still stick with the text label yeah, since it'd be a bit too special otherwise